packages/theme-tools/src/style-config/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { styleConfig } from "./single"
+import { multiStyleConfig } from "./multiple"
+import { Dict } from "@chakra-ui/utils"
+
+export const applyStyleConfigs = (theme: Dict) => {
+  Object.keys(theme.components).forEach((component) => {
+    const config = theme.components[component]
+
+    if (config.parts) {
+      theme.components[component] = multiStyleConfig(config)
+    } else {
+      theme.components[component] = styleConfig(config)
+    }
+  })
```

Comment:
`multiStyleConfig` and `styleConfig` are function that literally returns their arguments. I initially made them to provide style autocompletion and intellisense only.

That said, I'm not sure how useful the function above is.